huxley/api/views/delegate.py

```diff
@@ -24,11 +24,18 @@ def get_queryset(self):
         if school_id:
             queryset = queryset.filter(school_id=school_id)
 
+        committee_id = query_params.get('committee_id', None)
+        if committee_id:
+            queryset = queryset.filter(assignment__committee_id=committee_id)
+
         return queryset
 
+    def patch(self, request, *args, **kwargs):
```

Comment:
This method is defined in `ListUpdateModelMixin`. No need to define it here since it's inherited.